DateTimeCompareValidator.php

```diff
@@ -62,7 +62,11 @@ class DateTimeCompareValidator extends Validator
      * - `{compareAttribute}`: the label of the attribute to be compared with
      */
     public $message;
-
+    /**
+     * @var string Date format according to DateTime::createFromFormat specification.
+     *
```

Comment:
It's still there when I look at https://github.com/matias-yii/yii2-datetime-compare/blob/master/DateTimeCompareValidator.php

Maybe your editor is being nice and hiding it?